gecode/plan.sh

```diff
@@ -0,0 +1,26 @@
+pkg_name=gecode
+pkg_distname=$pkg_name
+pkg_origin=core
+pkg_version=3.7.3
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('gecode')
+pkg_source=http://www.gecode.org/download/gecode-${pkg_version}.tar.gz
+pkg_shasum=e7cc8bcc18b49195fef0544061bdd2e484a1240923e4e85fa39e8d6bb492854c
+pkg_dirname=${pkg_distname}-${pkg_version}
+pkg_deps=(core/glibc core/gcc)
+pkg_build_deps=(core/make core/gcc core/coreutils core/diffutils core/perl)
+
+do_build() {
+  ./configure \
+           --prefix=$pkg_prefix \
+           --disable-doc-dot \
+           --disable-doc-search \
+           --disable-doc-tagfile \
+           --disable-doc-chm \
+           --disable-doc-docset \
+           --disable-qt \
+           --disable-examples
+
+  make -j $(nproc)
+  make install
```

Comment:
I think you can remove this because it is part of the default `do_install()` step.